exchanges.yml

```
@@ -217,17 +217,62 @@ components:
                     type: string
                   template:
                       type: string
-            workflowId:
+            id:
               type: string
-              description: the ID that will be used for the created workflow. workflowId is OPTIONAL.
+              description: The ID that will be used for the created workflow. Passing an ID is OPTIONAL.
     CreateWorkflowResponse:
       type: object
       additionalProperties: false
-      description: Object containing information about a created workflow.
+      description: Response containing the workflowData Object created.
       properties:
-        workflowId:
-          type: string
-          description: The URL that uniquely identifies the created workflow.
+        workflowData:
```

Hmm, I think just the workflow `config` should be returned here and it looks like this is something a little different with some additional metadata is here. Certainly an implementation could do more, but I think we should only say that the workflow config is returned (i.e,. the same thing you'd get if you did a GET on the workflow ID).

```suggestion
        config:
```

---

This is a copy of what is returned by the [GET workflowId](https://w3c-ccg.github.io/vc-api/#get-workflow-configuration) as you are suggesting with the addition of the workflowId itself under the 'id' property as mentioned in the original issue.

---

Oh, I see we have a couple of mistakes in what's in the spec now then. There should be no `workflowData` property when creating a workflow, rather, the properties here:

https://github.com/w3c-ccg/vc-api/blob/5efdafd6f096efd6160f97a69448208c9297baf1/exchanges.yml#L190-L222

Should just be the main top-level properties in the payload, this is the "config" for the workflow.

I also see that the `GET workflowId` section is inconsistent and adds that extra `stepInformation` bit -- which we should remove (collapse it down so its nested properties are moved up a level so it matches what should be in the above payload for creating the workflow).